locations/spiders/eat_salad_fr.py

```
@@ -0,0 +1,21 @@
+
+from scrapy.http import TextResponse
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class EatSaladFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "eat_salad_fr"
+    item_attributes = {"brand": "Eat Salad", "brand_wikidata": "Q141214255"}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)